src/main/java/org/apache/sling/installer/factories/configuration/impl/ConfigTaskCreator.java

```diff
@@ -131,10 +131,11 @@ public InstallTask createTask(final TaskResourceGroup group) {
     @Override
     public void configurationEvent(final ConfigurationEvent event) {
         synchronized ( Coordinator.SHARED ) {
+            final String id = ConfigUtil.getPid(event);
             if ( event.getType() == ConfigurationEvent.CM_DELETED ) {
                 final Coordinator.Operation op = Coordinator.SHARED.get(event.getPid(), event.getFactoryPid(), true);
```

Comment:
It would be good to document somewhere by we pass the original PID and Factory PID to the coordinator but the recalculated one elsewhere.